vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -14,20 +14,22 @@
 )
 
 
-def send_sms_and_email_notifications_if_applicable(reference_number):
-    first_name = get_answer_from_form(["name", "first_name"])
-    last_name = get_answer_from_form(["name", "last_name"])
-    phone_number = get_answer_from_form(["contact_details", "phone_number_texts"])
-    email = get_answer_from_form(["contact_details", "email"])
-    contact_gp = should_contact_gp()
-
-    if phone_number:
-        govuk_notify_client.try_send_confirmation_sms(
-            phone_number, first_name, last_name, contact_gp
+def send_spl_failure_email_if_applicable(reference_number):
```

Comment:
i don't think in the _init.py file there is any env for the template id's - i know its not in this change log but they will need adding as currently in staging it blows up as the template is is never read in as in env var

Reply:
i have added all the env vars into master now so if you do a rebase then you should get all the template ids @danwilliams